ipwb/replay.py

```diff
@@ -153,6 +153,12 @@ def commandDaemon(cmd):
         return Response('bad command!')
 
 
+@app.route('/memento/*/')
+def showMementosForURIRs_sansJS():
+    urir = request.args.get('url')
+    return redirect('/memento/*/' + urir, code=301)
```

Comment:
I think a `302` status code would be more appropriate here, but I am not too sure about my rationale. Perhaps because we do not want to suggest the client to not use this form of URI again for the query.

Reply:
301 seems to say better than 302, "Use this other URI instead for now on."

Reply:
Yes, but that has an implicit assumption that the client should have some memory. Also, each redirect will only instruct about one query at a time rather than an entire class of pattern. That said, I am not too confident about my rationale here.